.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,262 @@
+name: Build and Test Phalcon
```

Comment:
This will be a badge text. For example:

```yaml
name: Linux CI
name: Windows CI
```

![image](https://user-images.githubusercontent.com/1256298/72644518-e3915500-3979-11ea-9bfa-5c594e4a6fc1.png)